### ISSUE TITLE
Not generating quotes around text

### ISSUE THREAD

**Issue by mortis304** [JAMFSupport/jHelper-GUI#8]
This tool doesn't seem to create quotes around the -title, -description, -heading, etc

#!/bin/bash
"/Library/Application Support/JAMF/bin/jamfHelper.app/Contents/MacOS/jamfHelper" -windowType utility -title This is a test title -description This is a test description -icon /Users/user/Desktop/logo.png -button1 OK -lockHUD

I can manually add "around my text" within the jHelper GUI and it works but shouldn't it automatically add them?

**Comment by kitzy** [comment]
I agree. Including special characters in the text could cause the generated command to fail. Strings should be quoted.

**Comment by canalnoises** [comment]
Yeah, this needs to be fixed...
